taglibs/async/await-tag-transformer.js

```diff
@@ -3,27 +3,36 @@
 var isObjectEmpty = require('raptor-util/isObjectEmpty');
 
 module.exports = function transform(el, context) {
-    var varName = el.getAttributeValue('var');
-    if (varName) {
-        if (varName.type !== 'Literal' || typeof varName.value !== 'string') {
-            context.addError(el, 'The "var" attribute value should be a string');
-            return;
-        }
+    if(!el.argument) {
+        context.addError(el, 'Invalid <await> tag. Argument is missing. Example: <await(user from data.userProvider)>');
+        return;
+    }
 
-        varName = varName.value;
+    var parts = el.argument.split(' from ');
```

Comment:
For some reason I was thinking that split with a string argument would only split on the first instance it found, but that's definitely not the case.  Once we figure out `as` vs `from` I'll change it to use a rooted regex.